src/app/services/authorization.service.ts

```diff
@@ -96,6 +96,10 @@ export class AuthorizationService {
     return this.canAccess(Entity.ASSET, Action.UPDATE);
   }
 
+  public canCreateAsset(): boolean {
```

Comment:
No need for this function since we are getting the create auth flag in the list.